SignalModelInterpolation/create_signal_ws_new_cat_2d_res_bkg.py

```diff
@@ -19,7 +19,7 @@ def unique(a, b):
 
 def getNuisanceDatacardName(name, year):
   if name == "fnuf":
-    return "CMS_hgg_nuisance_funf_13TeVscaleCorr"
+    return "CMS_hgg_nuisance_fnuf_13TeVscaleCorr"
```

Comment:
What is the difference?

Reply:
I don't see the difference... And I copied the line and manually substitute it, there is no change...